cli-copy.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+/**************************************************************************************************
+ * Imports
+ **************************************************************************************************/
+
+const commands      = require("./_modules/commands");
+const echo          = require("./_modules/echo");
+const program       = require("commander");
+const shell         = require("shelljs");
+
+
+/**************************************************************************************************
+ * Commands
+ **************************************************************************************************/
+
+// #region NuGet commands
+
+const copy = {
+    description() {
+        return "Copies files and/ or directories";
+    },
+    run(source, destination, options) {
```

Comment:
Consider adding a JSDoc comment regarding what `source`, `destination`, and `options` are